api/api-rule-violations.list

```diff
@@ -243,6 +243,8 @@ API rule violation: names_match,kubevirt.io/client-go/api/v1,KubeVirtConfigurati
 API rule violation: names_match,kubevirt.io/client-go/api/v1,KubeVirtConfiguration,SMBIOSConfig
 API rule violation: names_match,kubevirt.io/client-go/api/v1,KubeVirtSpec,CertificateRotationStrategy
 API rule violation: names_match,kubevirt.io/client-go/api/v1,LunTarget,ReadOnly
+API rule violation: names_match,kubevirt.io/client-go/api/v1,Memory,LockGuest
+API rule violation: names_match,kubevirt.io/client-go/api/v1,Memory,SharePages
```

Comment:
we need to avoid adding new violations. it's super simple to fix. i pointed out the issue in schema.go.

Reply:
Got it. I will remove the violations.